version/news/NEWS-2024.04.0-rel-chocolate-cosmos.md

```diff
@@ -36,6 +36,8 @@
 - RStudio now prints a warning if an overly-long line of console input is truncated when sending to R (#14420)
 - Updated zlib to 1.3.1 (rstudio/rstudio-pro#5971)
 - Vim: Fixed an issue where bracket highlight followed the bracket behind the cursor when in normal mode (#4152)
+- Updated nginx to 1.25.4 (rstudio/rstudio-pro#5970)
```

Comment:
Git is picking up some changes to this file that are unrelated to your changes. I think if you pull main into your branch it should take care of them?

Reply:
Pull `main` and not `rel-chocolate-cosmos`?

Reply:
Ah - it looks like although my local branch was against `rel-chocolate-cosmos` - I failed to change the comparison branch on the PR here. Gets me everytime...